QuickTCGA/Logic/TCGAUtilities.h

```diff
@@ -23,6 +23,8 @@
 
 #include <csignal>
 
+#include "vtkSlicerQuickTCGAModuleLogicExport.h"
```

Comment:
This is a standard file configured in the build directory of all Slicer libraries.